lib/audit_test.pm

```diff
@@ -73,12 +73,10 @@ sub upload_audit_test_logs {
     my ($testcase) = @_;
     upload_logs("$current_file");
 
-    # For syscall test case, different arch has different baseline file,
+    # For some test cases, different arch has different baseline file,
     # so we need to upload the correct file
-    if ($testcase eq 'syscalls') {
```

Comment:
Since you touched syscalls could you please offer the verification runs for syscalls.

Reply:
I did the verification runs for syscalls, but didn't paste the result.  Pasted it in the description.